docker-compose-test.yml

```
@@ -10,5 +10,22 @@ services:
     ports:
       - "9091:8080"
 
+  gotenberg:
+    image: gotenberg/gotenberg:7.7.0
+    networks:
+      - hmpps_int
+    container_name: gotenberg_int
+    ports:
+      - "3001:3001"
```

Can we change the port to 3100? As we are using 3001 in our Dockerfile.
```suggestion
      - "3100:3100"
```